src/io/flutter/performance/FlutterPerformanceView.java

```diff
@@ -250,57 +260,19 @@ FlutterViewAction registerAction(FlutterViewAction action) {
     return action;
   }
 
-  private void addFPSTab(JBRunnerTabs runnerTabs,
-                         FlutterApp app,
-                         ToolWindow toolWindow,
-                         boolean selectedTab) {
-    final PerfFPSTab perfTab = new PerfFPSTab(runnerTabs, app, toolWindow);
-    final TabInfo tabInfo = new TabInfo(perfTab)
-      .append(PERFORMANCE_TAB_LABEL, SimpleTextAttributes.REGULAR_ATTRIBUTES);
-    runnerTabs.addTab(tabInfo);
-    if (selectedTab) {
-      runnerTabs.select(tabInfo, false);
-    }
+  private void addFPSTab(JComponent parentPanel, @NotNull FlutterApp app, @NotNull Disposable parentDisposable) {
```

Comment:
should these methods be renamed now that they are no longer "tabs"?

Reply:
I removed them - they're now in-lined into their calling locations.